liquidctl/driver/nvidia.py

```diff
@@ -41,24 +51,16 @@ class EvgaPascal(SmbusDriver):
     _PERSIST = 0xe5
 
     @unique
-    class Mode(bytes, Enum):
-        def __new__(cls, value, required_colors):
-            obj = bytes.__new__(cls, [value])
-            obj._value_ = value
-            obj.required_colors = required_colors
-            return obj
-
+    class Mode(_ModeEnum):
         OFF = (0x00, 0)
         FIXED = (0x01, 1)
         RAINBOW = (0x02, 0)
```

Comment:
I noticed for your GPU there is no mode `0x03` or `0x04`, is it possible that those exist and are a fade / flash mode? It seems odd that they just skipped a couple numbers

Reply:
I though so too, but there doesn't seem to be nothing there.  Trying to fill in this gap is also why before I had these encoded in bits.